pkg/cmd/cli/describe/printer.go

```diff
@@ -1131,3 +1139,34 @@ func printRoleBindingRestrictionList(list *authorizationapi.RoleBindingRestricti
 	}
 	return nil
 }
+
+func printPspReview(pspreview *securityapi.PodSecurityPolicyReview, w io.Writer, options kctl.PrintOptions) error {
+	for _, allowedSA := range pspreview.Status.AllowedServiceAccounts {
```

Comment:
If we're not allowed, we need to say so. This applies to all three `printPsp*` methods.